lively.project/project.js

```diff
@@ -218,7 +218,17 @@ export class Project {
     loadedProject.url = url;
     loadedProject.configFile = resource(address.join('package.json').url);
     if (!onlyLoadNotOpen) {
+
       loadedProject.gitResource = await resource('git/' + await defaultDirectory()).join('..').join('local_projects').join(fullName).withRelativePartsResolved().asDirectory();
+      if (await loadedProject.gitResource.hasRemote()) {
+          const remoteURL = await loadedProject.gitResource.getRemote();
+          const userTokenInRemoteURL = remoteURL.match(/(gho_.*)@/)[1];
+          if (userTokenInRemoteURL !== currentUserToken()) {
+            const repoOwner = fullName.replace(/--.*/, '');
+            const name = fullName.replace(/.*--/, '');
+            await loadedProject.gitResource.changeRemoteURLToUseCurrentToken(currentUserToken(), repoOwner, name);
```

Comment:
This here is able to introduce a null user token if we are not logged in at all. This will make the project no longer loadable due to the line above.